lib/src/browser.dart

```diff
@@ -89,11 +89,15 @@ class _Chrome extends Browser {
   static Version _getVersion(NavigatorProvider navigator) {
     Match match = new RegExp(r"Chrome/(\d+)\.(\d+)\.(\d+)\.(\d+)\s")
         .firstMatch(navigator.appVersion);
```

Comment:
fyi, Jay and I talked via hipchat and turns out it's not just the version format that's different – the entire UA string is different.